src/doc/src/reference/manifest.md

```diff
@@ -433,59 +433,11 @@ default-run = "a"
 
 ### The `[badges]` section
 
-[crates.io] can display various badges for build status, test coverage, etc. for
-each crate. All badges are optional.
-
-- The badges pertaining to build status that are currently available are
-  Appveyor, CircleCI, Cirrus CI, GitLab, Azure DevOps, Travis CI and Bitbucket
-  Pipelines.
-- Available badges pertaining to code test coverage are Codecov and Coveralls.
-- There are also maintenance-related badges based on isitmaintained.com
-  which state the issue resolution time, percent of open issues, and future
-  maintenance intentions.
-
-Most badge specifications require a `repository` key. It is expected to be in
-`user/repo` format.
+[crates.io] does not display various badges for build status, test coverage, etc. for
+each crate any more. Only the following fields will be shown.
```

Comment:
I would not start off with what badges *don't* do.  Perhaps start off with a description, such as:

```
Badges are small status pieces of status information about a package. They can
be displayed as small images on a package registry website. 
```

Then follow with a description of the badges that are supported by crates.io.  I would keep the description of the maintenance badges, and the link to isitmaintained.com, for example.

Then, at the bottom after the example, include a note for anyone who may be confused as to what changed:

> Note: [crates.io] previously displayed badges next to a crate on its
> website, but that functionality has been removed. Packages should place
> badges in its README file and set [the readme field](#the-readme-field) so
> that the entire README is displayed on [crates.io].

Also, I'm uncertain, but it looks like crates.io is not showing *any* badges anymore?  What is the state of the maintenance badges?  Why would someone set them?

Also, the format for the fields is not explained.  What is the form for the `repository` field? Is it a URL to a website? A git URL?

Reply:
Since the [maintenance status is moved into the gui](https://github.com/rust-lang/crates.io/issues/2437) maybe the whole section should be removed?